### PR TITLE
Issue3016 text and graphic errors

### LINKED ISSUE #3016: Typo and miscellaneous fixes
This issue is a collection of text or graphic errors I found and fixed when I was studying other models for my own tasks. None of them affect the model results.
List:

- `Buildings.Applications.DataCenters.ChillerCooled.Equipment.BaseClasses.PartialPumpParallel`: Corrected the description for the valve raise time where languages for pumps were used.
- `Buildings.Applications.BaseClasses.Equipment.Validation.ElectricChillerParallel`: Empty subplots removed.
- `Buildings.Fluid.Storage.Examples.StratifiedUnloadAtMinimumTemperature`: Variable names and descriptions for the "top", "middle", and "bottom" were mixed. This is corrected.
- `Buildings.Fluid.Interfaces.PartialTwoPortTransport`: Corrected a typo in documentation.
- Classes under `Buildings.Controls.OBC.CDL.Conversions`: "Real", "Integer", and "Boolean" were mixed in the variable descriptions and in documentation. This is corrected. Also corrected a few typos.
- `Buildings.Fluid.Examples.ResistanceVolumeFlowReversal`: Corrected a typo.

### PR DESCRIPTION
**Pull request by hcasperfu**
This closes #3016.

### CODE REVIEW

File: Buildings/Controls/OBC/CDL/Conversions/BooleanToInteger.mo [hunk 1]
```diff
@@ -22,8 +22,8 @@ equation
     Documentation(
       info="<html>
 <p>
-Block that outputs the <code>Boolean</code>
-equivalent of the <code>Integer</code> input.
+Block that outputs the <code>Integer</code>
+equivalent of the <code>Boolean</code> input.
 </p>
 <pre>
   y = if u then integerTrue else integerFalse;
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Conversions/BooleanToInteger.mo:29]: Please also add this in the revision notes of this model, and in the revision notes of `Buildings/package.mo`.

File: Buildings/Controls/OBC/CDL/Conversions/BooleanToReal.mo [hunk 2]
```diff
@@ -22,16 +22,16 @@ equation
...
 </pre>
 <p>
 where <code>u</code> is of <code>Boolean</code> and <code>y</code>
-of <code>Integer</code> type,
-and <code>integerTrue</code> and <code>integerFalse</code> are parameters.
+of <code>Real</code> type,
+and <code>realTrue</code> and <code>realFalse</code> are parameters.
 </p>
 </html>",
       revisions="<html>
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Conversions/BooleanToReal.mo:37]: Please also add this in the revision notes of this model, and in the revision notes of `Buildings/package.mo`.

File: Buildings/Controls/OBC/CDL/Conversions/Validation/RealToInteger.mo [hunk 3]
```diff
@@ -13,7 +13,7 @@ model RealToInteger
 
 equation
   connect(ramp1.y,reaToInt.u)
-    annotation (Line(points={{-39,0},{28,0}},color={0,0,127}));
+    annotation (Line(points={{-38,0},{28,0}},color={0,0,127}));
   annotation (
```
Comment (mwetter) [inline Buildings/Controls/OBC/CDL/Conversions/Validation/RealToInteger.mo:17]: This change can be reverted.
You can run `git checkout master Buildings/Controls/OBC/CDL/Conversions/Validation/RealToInteger.mo` to restore to the former version.

**Review by mwetter — CHANGES_REQUESTED** [review]
Thanks for catching these errors. Can you please address the inline comments, and also
- make a PR for the changes in the `OBC` package for the `maint_8.1.x` branch (also update the release notes in `Buildings/package.mo`)
- make a PR for the IBPSA library for the changes in the `Fluid` package (as these would otherwise be overridden by the next merge).

File: Buildings/Controls/OBC/CDL/Conversions/BooleanToInteger.mo (hunk 1 above)
Comment (hcasperfu) [reply Buildings/Controls/OBC/CDL/Conversions/BooleanToInteger.mo:29]: Revision and release notes added in 8065511.

File: Buildings/Controls/OBC/CDL/Conversions/BooleanToReal.mo (hunk 2 above)
Comment (hcasperfu) [reply Buildings/Controls/OBC/CDL/Conversions/BooleanToReal.mo:37]: Revision and release notes added in 8065511.

**Pull request merged**